docs/unique/eidolon-field-abstract-model.md

```
@@ -887,185 +887,185 @@
+- [[heartbeat-fragment-demo]]
+- [[docs/unique/eidolon-field-math-foundations|eidolon-field-math-foundations]]
+- [[eidolon-node-lifecycle]]
+- [[eidolon-field-optimization]]
+- [[field-node-diagram-set]]
+- [[event-bus-projections-architecture|Event Bus Projections Architecture]]
 - [Simulation Demo](chunks/simulation-demo.md)
-- [aionian-circuit-math](aionian-circuit-math.md)
+- [[docs/unique/aionian-circuit-math|aionian-circuit-math]]
 - [Math Fundamentals](chunks/math-fundamentals.md)
 
 ## Sources
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)